### ISSUE TITLE
Feed Icon Link

### ISSUE THREAD

**Issue by ttepasse** [brentsimmons/JSONFeed#41]
The current example for linking one's feed with the JSON feed icon is missing an `alt` text. Since there is no neighbouring text link that makes it somewhat invisible to assistive technology. An example with best practice should have an alt text, I feel.

As an aside the link could also have an link relationship of `rel=alternate` and a media type of `application/feed+json`, if that's the media type used. Autodiscovery should detect this link, shouldn't it?

**Comment by voxpelli** [comment]
A standard way of providing extended metadata info for a link would tie well into the providing of a generic `links` key as well, as I suggested in #44.

As you say, one may want to provide a description as well as a media-type etc. and it's also common to provide eg. a `hreflang` on some links when one have multiple alternatives in different languages. JSON Feed should really support all classic metadata for links – for accessibility, extensibility and internationalization reasons.

**Comment by sukiletxe** [comment]
I confirm that an alt text would help. (Also, this is just providing an example and it wouldn't change the spec substantially, do you want me to do it?)

Right now, my screen readers would speak:

1. The feed's url, followed by "graphic link", or vice versa (NVDA on Windows); or
2. "Link, image" followed by a pause, followed by the feed's title (if there is any).

The first is way too verbose, the second way quiet.

**Comment by ad732** [comment]
If I understand correctly, you are referring to the one in the specification?

I think a better version would be:
```html
<a
  href="https://example.org/blog/feed.json"
  type="application/feed+json"
  title="Subscribe to 'My Blog' via JSON Feed"
  aria-label="Subscribe to 'My Blog' via JSON Feed"
>
  <img
    width="32"
    height="32"
    src="https://jsonfeed.org/graphics/icon.png"
    alt=""
    aria-hidden="true"
  />
</a>
```

I don't know if it should be used for auto discovery. [RSS Autodiscovery Specification](https://www.rssboard.org/rss-autodiscovery) does only specify the use of `<link>` elements for discovery.
